src/python/pants/backend/python/goals/export.py

```diff
@@ -370,7 +372,19 @@ async def export_virtualenvs(
         )
         return ExportResults(mv.result for mv in maybe_venvs if mv.result is not None)
 
-    # TODO: Deprecate this entire codepath.
+    # TODO: After the deprecation exipres, everything in this function below this comment
+    #  can be deleted.
+    warn_or_error(
+        "2.23.0.dev0",
+        "exporting resolves without using the --resolve option",
+        textwrap.dedent(
```

Comment:
I think softwrap is the universal thing to use now.

Reply:
For some reason that didn't look right when rendered. I'll fiddle with it in a future change, doesn't seem worth a CI round.

Reply:
Yeah, we use softwrap because it adjusts to the users' terminal size dynamically, e.g. huge 500 character columns. textwrap is instead based on the length of the source code.